### PR TITLE
Add mounts opts

### PR DESCRIPTION
**Pull request by calvn**
Addresses https://github.com/ory/dockertest/issues/75

### CODE REVIEW

**Comment by calvn** [comment]
Noticed that the error handling is flawed. I'll try to get a fix in, but not near my workstation at the moment.

**Comment by calvn** [comment]
I made the updates to error handling, so it should be good to go now. I also changed `RunWithOptions()` to accept a pointer to `RunOptions` instead to avoid unnecessary allocations. I've noticed this pattern on many libraries (see [go-github](https://github.com/google/go-github/blob/master/github/activity_events.go), for example). However, this might break any existing implementation that imports this particular (albeit unreleased) version, so that's one point to consider.

Let me know if you need to make updates, but I'll have very limited availability during the upcoming two weeks or so. If this is blocking you, feel free to amend any changes directly.

File: dockertest_test.go [hunk 1]
```diff
@@ -76,7 +77,7 @@ func TestMongo(t *testing.T) {
 		}
 
 		if response.StatusCode != http.StatusOK {
-			return fmt.Errorf("Could not connect to resource.")
+			return fmt.Errorf("could not connect to resource")
```
Comment (aeneasr) [inline dockertest_test.go:80]: While lower case starting characters for sentences are actually the way to go in Go, adding a dot at the end of the string is not :) But it doesn't really matter here, just dropping my 2 cents :)

**Comment by aeneasr** [comment]
Thank you for your contribution! Please [sign off](https://github.com/ory/dockertest/blob/v3/CONTRIBUTING.md#developers-certificate-of-origin) your commit using `git commit --amend -s`

File: dockertest_test.go (hunk 1 above)
Comment (calvn) [reply dockertest_test.go:80]: I removed the dot in this case, did not add one :)

**Comment by aeneasr** [comment]
Sorry, one more thing, would you find the time to introduce this new function to the [examples](https://github.com/ory/dockertest/tree/v3/examples)? otherwise people will probably never find this functionality.

**Comment by aeneasr** [comment]
Perfec,t thank you!

**Comment by calvn** [comment]
Is there a planned release (minor/patch) for v3 that will have `RunWithOptions()` in the near future? I would like to use it, but have the package be imported from gopkg.in. Thanks!

**Comment by aeneasr** [comment]
done!

**Pull request merged**